src/main/resources/phrases_en_US.properties

```diff
@@ -36,6 +36,8 @@ QnMerge.checks-are-failed=Can't merge it. Some CI checks were failed. \
                           Apparently, the pull request is not ready to be \
                           merged since it has some problems. \
                           Please, fix them first.
+QnMerge.system-files-affected=Can't merge it. File `.rultor.yml` is affected by \
+                              this pull request.
```

Comment:
@pnatashap would be great if we can explain to the user, why we don't accept modifications to this particular file. Let's say: "Can't merge this pull requests, because the file `.rultor.yml` was modified. This is a security threat. Make a separate pull request with the modification of this particular file and ask project architect to merge it manually."

Reply:
@pnatashap also, I think it's better not to encode the name of the file here, but use `%s` and pass the name of it as a parameter to `String.format`